File: CPP_Joc_Windows_Android/SH_Client_Win_Cpp_Cmake/App/src/rpg3D/gw/entity/module/TeamModule.cpp
#include "TeamModule.h"

using namespace rpg3D;

TeamModule::TeamModule(
    IWorldEntity* entity,
    TeamTemplate* t)
    : super(t->moduleName, entity),
      t(t)
{
    //void
}

int TeamModule::getTeamIndex() {
	return teamIndex;
}

void TeamModule::setTeamIndex(int teamIndex) {
	this->teamIndex = teamIndex;
}

TeamModule::~TeamModule() {
	//void
}

File: NodeJs_TypeScript_JavaScript_Server/Project_NodeJs_VSCode/src/squareheads/adventlandia/basiczone/gwi/entity/module/team/TeamModule.ts
import { TeamModuleSnapshotDelta } from "./TeamModuleSnapshotDelta";
import { WorldEntityLocal } from "../../WorldEntityLocal";
import { TeamTemplate } from "../../template/TeamTemplate";

export class TeamModule extends TeamModuleSnapshotDelta {

  public constructor(entity: WorldEntityLocal, template: TeamTemplate) {
    super(entity, template);

    //void
  }

}
